tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -47,7 +49,12 @@ preferably, RStudio loaded on your computer.
 
 ### Data to Download
 
-{% include/dataSubsets/_data_Imaging-Spec-Data-H5.html %}
+{% include/dataSubsets/_data_Imaging-Spec-Data-H5-2020.html %}
+
+*Remember* that the example dataset included here only has 1 out of every 4 bands
```

Comment:
I'd also add mention of this into the intro and say something like, if you are using a full NEON mosaic with 426 bands you'll need to adjust the band #s.

Reply:
Good idea - I added this information to the introduction after it mentions that the full dataset has 426 bands.